src/com/clover/remote/client/transport/websocket/WebSocketCloverTransport.ts

```diff
@@ -25,6 +25,24 @@ export abstract class WebSocketCloverTransport extends CloverTransport implement
 
 	private reconnectDelay: number = 3000;
 
+    /**
+	 * We do not wantto start up multiple reconnect threads.  This should alleviate that
+	 * @type {boolean}
+     */
+	private reconnecting: boolean = false;
+
+	/**
+	 * Subclasses need to set this at times.
+	 *
+	 * @param newValue
+	 * @returns {boolean}
+     */
+	protected setReconnecting(newValue: boolean): boolean {
```

Comment:
Why is the return value the old value?

Reply:
Doesn't appear the callers ever use the return ...